contribs/src/main/java/com/netflix/conductor/contribs/publisher/TaskStatusPublisher.java

```diff
@@ -46,7 +46,8 @@ public void run(){
                 try {
                     task = blockingQueue.take();
                     taskNotification = new TaskNotification(task);
-                    LOGGER.info("Start Publishing TaskNotification: taskId {} taskName {}", task.getTaskId(),task.getTaskDefName());
+                    String jsonTask = taskNotification.toJsonStringWithoutInputOutput();
+                    LOGGER.info("Start Publishing TaskNotification: {}", jsonTask);
```

Comment:
reword this comment to "Publishing TaskNotification"

Reply:
fixed